sdlib/d/gc/tcache.d

```diff
@@ -127,6 +173,7 @@ public:
 		if (isLargeSize(size)) {
 			auto npd = getPageDescriptor(newPtr);
 			npd.extent.setUsedCapacity(size);
+			npd.extent.setFinalizer(finalizer);
```

Comment:
Don't do finalization magic in realloc. this is not necessary at this time, and this is untested anyways.